packages/dom/src/dom/is-text-field.js

```diff
@@ -26,6 +26,7 @@ export default function isTextField( node ) {
 		'reset',
 		'submit',
 		'number',
+		'email',
```

Comment:
Could any component by negatively affected by this spec correction?

Reply:
Also, I really dislike `isTextField`'s exclusion approach. Since, in fact, _most_ input types are not text fields, we might be better off with a stricter inclusion approach, i.e. `[ 'text', ... ].includes( type )`.

Reply:
I don't think anything would be negatively affected by this.. And I also think a reversed approach would be okay, but we would really need to make sure we have all the needed types 😄